### ISSUE TITLE
errors compiling

### ISSUE THREAD

**Issue by v1k0d3n** [ggreer/dsniff#2]
./configure: line 410: test: pcap-bpf.h: integer expression expected
configure: error: pcap.h
./configure: line 299: return: pcap-bpf.h: numeric argument required
./configure: line 309: exit: pcap-bpf.h: numeric argument required

seems like i get this issue too. when i try to brew libpcap it states that it's already included with OSX:

Error: No available formula for libpcap 
Apple distributes libpcap with OS X, you can find it in /usr/lib.

if I compile with the /usr/lib directory, i get the error above. has anyone seen this happen on their platform as well? what's the best work around for this issue (i have tried compiling with Xcode's libpcap as well, with same issue)...

**Comment by fr3nch13** [comment]
I'm having the same errors. I'm compiling on OS X 10.11

checking for libpcap... /usr/local/opt/libpcap
./configure: line 410: test: pcap-bpf.h: integer expression expected
configure: error: pcap.h
./configure: line 299: return: pcap-bpf.h: numeric argument required
./configure: line 309: exit: pcap-bpf.h: numeric argument required

As v1k0d3n commented about libpcap, I also tried Apple's copy, and to the same error as well:

checking for libpcap... /usr/lib/libpcap.dylib
./configure: line 410: test: pcap-bpf.h: integer expression expected
configure: error: pcap.h
./configure: line 299: return: pcap-bpf.h: numeric argument required
./configure: line 309: exit: pcap-bpf.h: numeric argument required

I was able to get it to compile by dropping the pcap line: 
./configure \
  --with-openssl=/usr/local/opt/openssl \
  --with-libnet=/usr/local/opt/libnet \
  --with-libnids=/usr/local/opt/libnids \
  --without-x

Currently, I'm getting this: 
$ ./mailsnarf 
libnet_open_raw4(): SOCK_RAW allocation failed: Operation not permitted

mailsnarf: Bad file descriptor

Update: I'm an idiot. Run it with sudo.
https://trac.macports.org/ticket/38049#comment:3

**Comment by v1k0d3n** [comment]
@fr3nch13 so it all works for you now?

**Comment by fr3nch13** [comment]
Yeah, I just wanted to share what I did in case others have the same issue.
